### ISSUE TITLE
instructions unclear

### ISSUE THREAD

**Issue by MaximilianKohler** [jibsen/pngquant-winbuild#6]
In a Windows 10 powershell window I entered your first line of code:

```
git submodule update --init --recursive
fatal: not a git repository (or any of the parent directories): .git
```
I have no idea what the next line means:
`And then build with mingw-w64 (make).`

Your instructions are referenced here https://squidfunk.github.io/mkdocs-material/setup/dependencies/image-processing/#windows_1 - I did the previous steps without issue.

**Comment by jibsen** [comment]
You are right, the instructions in the README are not exactly detailed. The submodule command requires you to first clone the repository, something like

~~~.sh
git clone https://github.com/jibsen/pngquant-winbuild
cd pngquant-winbuild
git submodule update --init --recursive
~~~

This just downloads the source, you then have to build it using some kind of GCC compiler toolchain (for instance mingw-w64 or MSYS2), the make command may be called `mingw32-make`, `gmake`, or plain `make`.

The releases on this repository have precompiled Win64 binaries, and I believe the pngquant website has up-to-date binaries as well.

**Comment by MaximilianKohler** [comment]
>The releases on this repository have precompiled Win64 binaries, and I believe the pngquant website has up-to-date binaries as well.

Does that mean we can install the program on Windows with with the Win64 binary? Or it runs standalone with that binary and we somehow have to associate that binary with the program that needs to use it (_MKDocs in this case_)?

I checked the pngquant site https://pngquant.org/ and I see they have a `windows binary` under `Command-line`, but no instructions on how to install/use it. I downloaded theirs and checked the `readme` but it only has info on how to use it on its own; no info on how to use it with another program like MKDocs.

**Comment by jibsen** [comment]
Usually "installing" a command line tool consists of either placing it in a folder that is on the path, or modifying the path to include where the executable is placed. I realize this all appears somewhat archaic.

Another option is to install pngquant using a package manager (like Chocolatey or Scoop), which should take care of things like setting up the path.
